src/main/kotlin/com/hibob/project/dao/models.kt

```diff
@@ -0,0 +1,68 @@
+package com.hibob.project.dao
+
+import java.sql.Timestamp
+
+data class FeedbackConfiguration(
+    val id: Long,
+    val employeeId: Long,
+    val companyId: Long,
+    val text: String,
+    val creationTime: Timestamp,
+    val isAnonymous: Boolean,
+    val status: StatusType
+)
+
+data class FeedbackCreationRequest(
+    val text: String,
+    val creationTime: Timestamp,
+    val isAnonymous: Boolean,
+    val status: StatusType
+) {
+    fun toFeedbackConfiguration(id: Long, employeeId: Long, companyId: Long): FeedbackConfiguration {
```

Comment:
If this is only for tests, move it to the tests file. There shouldn't be code for tests in the code itself.

If you are planning to use it in general, then my comment is that I don't understand why you need this conversion method

Reply:
Got it, Ill move it to test